criu/parasite-syscall.c

```diff
@@ -179,8 +179,11 @@ int parasite_dump_thread_seized(struct parasite_thread_ctl *tctl,
 
 	pc = args->creds;
 	pc->cap_last_cap = kdat.last_cap;
-
+#ifdef CONFIG_MIPS
+	memcpy(tc->blk_sigset, compel_thread_sigmask(tctl), sizeof(k_rtsigset_t));
+#else
```

Comment:
Any reason why MIPS should be special?

Reply:
sizeof(k_rtsigset_t) is 16byte (two uint64_t) in MIPS platform. I set n_blk_sigset = 2 in pstree.c